python/lsst/analysis/tools/tasks/makeTractPostageStamp.py

```diff
@@ -0,0 +1,300 @@
+
+            imdata = copy.deepcopy(im)
+            imdata[mask] = np.nan
+            med = np.nanmedian(imdata)
+            mad = np.nanmedian(np.fabs(im - med))
+            vmin = med - 1 * mad
+            vmax = med + 15 * mad
+            ax.imshow(imdata, extent=Extent, vmin=vmin, vmax=vmax, cmap=cmap)
+
+            # Highlight regions of "NODATA".
+            imnodata = copy.deepcopy(im)
+            imnodata[~mask] = np.nan
+            ax.imshow(imnodata, extent=Extent, alpha=0.8, cmap=cmapred)
+
+            ax.annotate(
+                patchId,
+                (np.mean(Ras), np.mean(Decs)),
+                color="k",
+                ha="center",
+                va="center",
+                path_effects=[pathEffects.withStroke(linewidth=2, foreground="w")],
+            )
+
+        # Indicate blank patches as hatched regions
+        for patchId in emptyPatches:
+
+            Ras, Decs = self.getPatchCorners(tractInfo, patchId)
+
+            # Account for the RA wrapping using negative RA values.
+            if RaSpansZero:
+                Ras = [ra - 360 if ra > 180.0 else ra for ra in Ras]
+            Extent = (max(Ras), min(Ras), max(Decs), min(Decs))
+            ax.plot(
+                [min(Ras), max(Ras), max(Ras), min(Ras), min(Ras)],
+                [min(Decs), min(Decs), max(Decs), max(Decs), min(Decs)],
+                "k",
+                lw=0.5,
+            )
+
+            cs = ax.contourf(np.ones((10, 10)), 1, hatches=["xx"], extent=Extent, colors="none")
+            for c in cs.collections:
+                c.set_edgecolor("red")
+            ax.annotate(
+                patchId,
+                (np.mean(Ras), np.mean(Decs)),
+                color="k",
+                ha="center",
+                va="center",
+                path_effects=[pathEffects.withStroke(linewidth=2, foreground="w")],
+            )
+
+        # Draw axes around the entire tract:
+        ax.set_xlabel("R.A. (deg)", fontsize=20)
+        ax.set_ylabel("Dec. (deg)", fontsize=20)
+
+        tractCorners = self.getTractCorners(skymap, tractInfo.getId())
+
+        tractRas = [ra for (ra, dec) in tractCorners]
+        if RaSpansZero:
+            tractRas = [ra - 360.0 for ra in tractRas]
+        ax.set_xlim(max(tractRas), min(tractRas))
+        ticks = [t for t in ax.get_xticks() if t >= min(tractRas) and t <= max(tractRas)]
+        tickLabels = [f"{t % 360:.1f}" for t in ticks]
+        ax.set_xticks(ticks, tickLabels)
+
+        tractDecs = [dec for (ra, dec) in tractCorners]
+        ax.set_ylim(min(tractDecs), max(tractDecs))
+
+        ax.tick_params(axis="both", labelsize=10, length=0, pad=1.5)
+        ax.set_title(str(tract) + ": " + band, fontsize=20)
+        fig.canvas.draw()
+
+        return fig
+
+    def getTractCorners(self, skymap, tract):
+        """Calculate the corners of a tract, given  skymap.
+        ***This needs to be moved from here***
```

Comment:
Is this a reminder to yourself or a to do? If it is a To Do it should have a ticket number.

Reply:
Does this, or something similar, already exist in plotUtils?